### PR TITLE
Bybit: borrowMargin, repayMargin

### PR DESCRIPTION
**Pull request by Dan-krm**
Added borrowMargin and repayMargin to Bybit:

### borrowMargin:
```
bybit.borrowMargin (USDT, 100)
2022-07-19T04:08:08.264Z iteration 0 passed in 354 ms

{
  id: 9233,
  currency: 'USDT',
  amount: 100,
  symbol: undefined,
  timestamp: 1658203688264,
  datetime: '2022-07-19T04:08:08.264Z',
  info: {
    ret_code: '0',
    ret_msg: '',
    ext_code: null,
    ext_info: null,
    result: '9233'
  }
}
2022-07-19T04:08:08.264Z iteration 1 passed in 354 ms
```

### repayMargin:
```
bybit.repayMargin (USDT, 100.00048375)
2022-07-19T04:12:53.894Z iteration 0 passed in 279 ms

{
  id: 9285,
  currency: 'USDT',
  amount: 100.00048375,
  symbol: undefined,
  timestamp: 1658203973894,
  datetime: '2022-07-19T04:12:53.894Z',
  info: {
    ret_code: '0',
    ret_msg: '',
    ext_code: null,
    ext_info: null,
    result: '9285'
  }
}
2022-07-19T04:12:53.894Z iteration 1 passed in 279 ms
```

### CODE REVIEW

File: js/bybit.js [hunk 1]
```diff
@@ -4895,6 +4897,96 @@ module.exports = class bybit extends Exchange {
         };
     }
 
+    async borrowMargin (code, amount, symbol = undefined, params = {}) {
```
Comment (carlosmiei) [inline js/bybit.js:4900]: Can we add these related exceptions (currently are generic `ExchangeErrors`) ?

```Javascript
ccxt.base.errors.ExchangeError: bybit {"ret_code":-6017,"ret_msg":"Repayment amount has exceeded the total liability","ext_code":null,"ext_info":null,"result":null}

ccxt.base.errors.ExchangeError: bybit {"ret_code":-6026,"ret_msg":"Amount to borrow cannot be larger than the max. amount to borrow (per transaction)","ext_code":null,"ext_info":null,"result":null}

ccxt.base.errors.ExchangeError: bybit {"ret_code":-6025,"ret_msg":"Amount to borrow cannot be lower than the min. amount to borrow (per transaction)","ext_code":null,"ext_info":null,"result":null}

ccxt.base.errors.ExchangeError: bybit {"ret_code":-1103,"ret_msg":"An unknown parameter was sent.","ext_code":null,"ext_info":null,"result":null}

```

**Comment by carlosmiei** [comment]
LGTM! 🙂 
```
 python cli.py bybit borrowMargin "USDT" 100 --sandbox  
Python v3.9.13
CCXT v1.90.89
bybit.borrowMargin(USDT,100)
{'amount': 100,
 'currency': 'USDT',
 'datetime': '2022-07-20T10:57:23.990Z',
 'id': 4264,
 'info': {'ext_code': None,
          'ext_info': None,
          'result': '4264',
          'ret_code': '0',
          'ret_msg': ''},
 'symbol': None,
```

```
python cli.py bybit repayMargin "USDT" 100 --sandbox
Python v3.9.13
CCXT v1.90.89
bybit.repayMargin(USDT,100)
{'amount': 100,
 'currency': 'USDT',
 'datetime': '2022-07-20T10:57:43.978Z',
 'id': 2835,
 'info': {'ext_code': None,
          'ext_info': None,
          'result': '2835',
          'ret_code': '0',
          'ret_msg': ''},
 'symbol': None,
 'timestamp': 1658314663978}
```

File: js/bybit.js (hunk 1 above)
Comment (Dan-krm) [reply js/bybit.js:4904]: I've added these all as InvalidOrder exceptions

Comment (carlosmiei) [reply js/bybit.js:4904]: @Dan-krm nice, thanks

**Comment by carlosmiei** [comment]
LGTM ✅ ✅ 

```
 p bybit borrowMargin "USDT" 100 --sandbox  
Python v3.9.7
CCXT v1.91.51
bybit.borrowMargin(USDT,100)
{'amount': 100,
 'currency': 'USDT',
 'datetime': '2022-08-05T09:57:16.083Z',
 'id': 9228,
 'info': {'ext_code': None,
          'ext_info': None,
          'result': '9228',
          'ret_code': '0',
          'ret_msg': ''},
 'symbol': None,
 'timestamp': 1659693436083}
```
```
p bybit repayMargin "USDT" 100 --sandbox                                         
Python v3.9.7
CCXT v1.91.51
bybit.repayMargin(USDT,100)
{'amount': 100,
 'currency': 'USDT',
 'datetime': '2022-08-05T09:57:57.226Z',
 'id': 7553,
 'info': {'ext_code': None,
          'ext_info': None,
          'result': '7553',
          'ret_code': '0',
          'ret_msg': ''},
 'symbol': None,
 'timestamp': 1659693477226}
```

**Review by carlosmiei — APPROVED** [review]

**Comment by Dan-krm** [comment]
These methods were added with this PR: https://github.com/ccxt/ccxt/pull/14588

**Pull request closed without merging**